### PR TITLE
Send tickvalues

### LINKED ISSUE #97: [FEATURE] Send tickvalues
#### Send tickvalues from backend in order to set number of ticks and format
e.g. [15, 'd.'] -> [tickcount, format]

### LINKED ISSUE #111: [FEATURE] Fetch all data in MapContainer

### LINKED ISSUE #119: [FEATURE] Find fitting intervals for datapoints

### PR DESCRIPTION
**Pull request by mariusaarsnes**
Closes #97 
Closes #111 
Closes #119

### CODE REVIEW

File: webapp/src/components/Graph/DateGraph.tsx [hunk 1]
```diff
@@ -6,93 +6,20 @@ import Graph from './Graph';
...
-        .catch(error => {
-          console.log(error);
-        });
-    } else {
-      setDataSet([]);
-    }
-  }, [unitNumber, dates]);
-*/
+const DateGraph: FunctionComponent<DateGraphProps> = ({ dataSet, tickState }) => {
+  console.log(tickState);
```
Comment (alfredbirk) [inline webapp/src/components/Graph/DateGraph.tsx]: Remove console.log

File: api/lib/controllers/MobilityDataController.ts [hunk 2]
```diff
@@ -28,22 +29,83 @@ const SPLIT_INTERVAL_ENUM = {
   DAY: 5
 };
 
+const TICK_COUNT_ENUM = {
+  YEAR: 365,
+  HALFYEAR: 183,
+  QUARTERYEAR: 92,
+  MONTH: 31,
```
Comment (alfredbirk) [inline api/lib/controllers/MobilityDataController.ts]: All tickvalues should be max about 25. If more, they will start overlapping. I suggest: Set month, quorteryear, halfyear and year to 15.

File: webapp/src/components/Graph/DateGraph.tsx (hunk 1 above)
Comment (mariusaarsnes) [reply webapp/src/components/Graph/DateGraph.tsx]:
```suggestion
```

File: api/lib/controllers/MobilityDataController.ts [hunk 3]
```diff
@@ -28,22 +29,83 @@ const SPLIT_INTERVAL_ENUM = {
   DAY: 5
 };
 
+const TICK_COUNT_ENUM = {
+  YEAR: 365,
```
Comment (mariusaarsnes) [inline api/lib/controllers/MobilityDataController.ts]:
```suggestion
  YEAR: 12,
```

File: api/lib/controllers/MobilityDataController.ts [hunk 4]
```diff
@@ -28,22 +29,83 @@ const SPLIT_INTERVAL_ENUM = {
...
+  HALFYEAR: 183,
```
Comment (mariusaarsnes) [inline api/lib/controllers/MobilityDataController.ts]:
```suggestion
  HALFYEAR: 12,
```

File: api/lib/controllers/MobilityDataController.ts [hunk 5]
```diff
@@ -28,22 +29,83 @@ const SPLIT_INTERVAL_ENUM = {
...
+  QUARTERYEAR: 92,
```
Comment (mariusaarsnes) [inline api/lib/controllers/MobilityDataController.ts]:
```suggestion
  QUARTERYEAR: 6,
```

File: api/lib/controllers/MobilityDataController.ts (hunk 2 above)
Comment (mariusaarsnes) [inline api/lib/controllers/MobilityDataController.ts]:
```suggestion
  MONTH: 6,
```

**Pull request merged**